common/internal/test_util.go

```diff
@@ -0,0 +1,27 @@
+package internal
+
+import (
+	"context"
+
+	"github.com/anz-bank/pkg/log"
+	"github.com/anz-bank/sysl-go/logconfig"
+)
+
+type TestHook struct {
+	Entries []log.LogEntry
+}
+
+func (h *TestHook) OnLogged(entry *log.LogEntry) error {
+	h.Entries = append(h.Entries, *entry)
+	return nil
+}
+
+func NewTestContextWithLoggerHook() (context.Context, *TestHook) {
```

Comment:
This is largely duplicated from `common.test_util.go`. Can we unify these?

Reply:
Extract this to the testutil package, cannot keep this part in the common/test_util will get import cycle error.